src/components/10-atoms/checkbox/index.scss

```diff
@@ -127,90 +106,62 @@ axa-checkbox {
       }
     }
   }
-
   .a-checkbox__content {
     display: block;
     margin: -19px 0 0 30px;
     line-height: 1.37;
     color: $color-dark-grey;
-
     &--inverted {
       color: $color-prim-white;
     }
   }
-
   .a-checkbox__error {
     display: block;
     margin: $checkbox-error-top-margin 0 0 0;
     line-height: 1.37;
   }
-
   &[label]:not([label=""]) { // if label is set and is not empty the error text should have some margin-left
     .a-checkbox__error {
       margin: $checkbox-error-top-margin 0 0 ($checkbox-dimension + $checkbox-label-left-margin);
     }
   }
-
   &[disabled] {
     .a-checkbox__content {
       color: $color-light-grey;
-
       &--inverted {
         color: $color-disabled-inverted;
       }
     }
-
     .a-checkbox__input:checked + .a-checkbox__icon::after {
       border-color: $color-silver;
       background-color: $color-silver;
     }
   }
-
   &[error] {
```

Comment:
Why did you remove all line breaks here?

Reply:
idk, was prettier probably but I changed it back